src/gui/ogr/qgsogrhelperfunctions.cpp

```diff
@@ -269,7 +269,8 @@ QString createProtocolURI( const QString &type, const QString &url,  const QStri
   {
     uri = QStringLiteral( "DODS:%1" ).arg( url );
   }
-  else if ( type == QLatin1String( "WFS3" ) )
+  // Check beginning because of "experimental"
```

Comment:
can you explain better why should change in case of experimental?

Reply:
just because the match is done on the combo text (which is IMO wrong, but out of scope).